package.json

```diff
@@ -9,7 +9,8 @@
     "de.basisprofil.r4": "1.3.2",
     "hl7.fhir.r4.core": "4.0.1",
     "kbv.ita.for": "1.1.0",
-    "kbv.basis": "1.3.0",
-    "de.gematik.erezept-workflow.r4": "1.2.1"
+    "de.gematik.erezept-workflow.r4": "1.2.3",
+    "de.abda.eRezeptAbgabedatenpkv": "1.2.0",
+    "KBV.Basis": "1.3.0"
```

Comment:
Ich erinnere mich auch, dass es Probleme beim Auflösen der in GROSSBUCHSTABEN geschriebenen KBV-Pakete in der Pipeline gab. Bitte verstehe dies nur als Hinweis, falls Du auf ungewöhnliche Probleme beim der Veröffentlichung stößt.